client/testFile.js

```diff
@@ -1 +1,6 @@
-// testing if this file shows up in my branch
\ No newline at end of file
+// testing if this file shows up in my branch
+// #2 testing to see if this file shows up in my branch in github
+  // this time I will include a comment in the pull request review
+  // make the changes on my local
+  // make another pull request
+  // approve the pull request and merge the branch to the master
```

Comment:
Add an extra line to mention that you successfully reviewed your pull request and made the change by adding "MADE CHANGE!"